starting_package/openvino_workbench/arguments_parser.py

```diff
@@ -19,13 +19,33 @@
 
 import argparse
 import os
+import sys
 from typing import Optional
 
 
 def get_proxy_from_env(proxy: str) -> Optional[str]:
     return os.getenv(proxy) or os.getenv(proxy.upper())
 
 
+def validate_args_for_restart(args: argparse.Namespace, parser: argparse.ArgumentParser):
+    # If detached restart is needed then there should be exactly 4 arguments
+    # and '--detached' should be one of them
+    if args.detached and len(sys.argv) != 4:
```

Comment:
I think, we need to use `args ` to check

Reply:
Unfortunately, `argparse.Namespace` has nothing related to its length / how much arguments are passed.